Assets/TextMesh Pro/Fonts/LiberationSans - OFL.txt

```diff
@@ -0,0 +1,46 @@
+Digitized data copyright (c) 2010 Google Corporation
+	with Reserved Font Arimo, Tinos and Cousine.
```

Comment:
Think everything under this can be removed for a simpler font face but I don't really have one ready at the moment.